### PR TITLE
Add current period data to dashboard

### PR DESCRIPTION
**Pull request by vkrmis**
- Add current month cohort bar in the district and facility monthly report views
- Add current quarter bar in the district and facility quarterly report views
- Add current quarter data for the district and facility analytics tables

https://www.pivotaltracker.com/story/show/170280451

### CODE REVIEW

File: app/controllers/analytics/districts_controller.rb [hunk 1]
```diff
@@ -70,7 +70,7 @@ def set_dashboard_analytics(period, prev_periods)
       analytics_cache_key_dashboard(period),
       @organization_district.dashboard_analytics(period: period,
                                                  prev_periods: prev_periods,
-                                                 include_current_period: @show_current_month))
+                                                 include_current_period: @show_current_period))
```
Comment (govindkrjoshi) [inline app/controllers/analytics/districts_controller.rb]: Can we inline this?

File: app/controllers/analytics/facilities_controller.rb [hunk 2]
```diff
@@ -71,7 +71,7 @@ def set_dashboard_analytics(period, prev_periods)
       analytics_cache_key_dashboard(period),
       @facility.dashboard_analytics(period: period,
                                     prev_periods: prev_periods,
-                                    include_current_period: @show_current_month))
+                                    include_current_period: @show_current_period))
```
Comment (govindkrjoshi) [inline app/controllers/analytics/facilities_controller.rb]: Same here.

File: app/queries/cohort_analytics_query.rb [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
     @patients = patients
+    @include_current_period = include_current_period
   end
 
   def patient_counts_by_period(period, prev_periods, from_time: Time.current)
     results = {}
 
-    (0..(prev_periods - 1)).each do |periods_back|
+    # index is a quick hack to allow toggling the current period in the results.
+    index = @include_current_period ? -1 : 0
```
Comment (govindkrjoshi) [inline app/queries/cohort_analytics_query.rb]: This is not very obvious. Can we avoid this?

File: app/controllers/analytics/districts_controller.rb (hunk 1 above)
Comment (prabhanshuguptagit) [reply app/controllers/analytics/districts_controller.rb]: `show_current_period` needs to be declared, it's being used in the view (`_analytics_table.html.erb`)

**Pull request merged**